opengever/maintenance/scripts/migrate_and_deactivate_meetings.py

```diff
@@ -106,6 +110,53 @@ def replace_meeting_dossier_with_normal_dossier(self):
             # delete meeting_dossier
             api.content.delete(meeting_dossier)
 
+    def migrate_agendaitems_to_subdossiers(self):
+        """ We create a subdossier in the meeting dossier for each agendaitem
+        and copy or move its documents into that subdossier.
+        """
+        for meeting in Meeting.query:
+            meeting_dossier = meeting.get_dossier()
+            for agendaitem in meeting.agenda_items:
+                # create a subdossier
+                dossier = api.content.create(
+                    type='opengever.dossier.businesscasedossier',
+                    title=agendaitem.get_title(include_number=True, formatted=True),
+                    container=meeting_dossier)
+
+                # Copy documents from submitted proposal into subdossier
+                for doc in agendaitem.get_documents():
+                    self.copy_and_add_to_mapping(doc, dossier)
+
+                # Move excerpts
+                for doc in agendaitem.get_excerpt_documents():
+                    self.move_and_add_to_mapping(doc, dossier)
```

Comment:
Maybe add an `assert doc in meeting_dossier.objectValues()` here, to make sure we're only moving documents from the main (new) meeting dossier into one of its subdossiers, and not from anywhere else? (Which would indicate something unexpected is going on).